clarifai_model_utils/llm_eval/evaluator/harness_eval/doc/new_template.md

```diff
@@ -0,0 +1,184 @@
+There are 2 ways to add new task:
```

Comment:
Would it be accurate to say that to add a new task, we must specify configs, either via a yaml file or via the TaskConfig class. Both will be parsed into a TaskConfig object?